ludwig/schema/optimizers.py

```diff
@@ -50,6 +51,11 @@ class BaseOptimizerConfig(schema_utils.BaseMarshmallowConfig, ABC):
     a `ValidationError`.
     """
 
+    @property
+    def is_paged(self) -> bool:
```

Comment:
Is this needed?

Reply:
I only added it for the schema validation check, but it may not be necessary